portfolio/src/main/webapp/game.js

```diff
@@ -0,0 +1,65 @@
+var canvas = document.getElementById("game");
+var ctx = canvas.getContext("2d");
+var xPos = canvas.width / 2;
```

Comment:
Keep in mind all these top-level variables are being added to the global env and would be accessible/editable from any other js file.

I'd especially encourage you to reconsider having temporary x/y position be in the global state and try to refactor them as arguments to `drawBall`.